static/app/views/dashboards/widgetBuilder/components/sortBySelector.spec.tsx

```diff
@@ -110,7 +110,7 @@ describe('WidgetBuilderSortBySelector', function () {
     expect(mockNavigate).toHaveBeenLastCalledWith(
       expect.objectContaining({
         ...router.location,
-        query: expect.objectContaining({sort: ['-count']}),
```

Comment:
Why does this need to be changed?

Reply:
It was failing and `-count()` was the new result. I'm guessing this has to do with an sdk update?